mongodb:
  uri: mongodb://localhost:27017/
  config: /home/whi/whi_db/mongod.conf
  db_name: whi_db
